wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -438,13 +437,16 @@ application. For example, here's a typical error which might be issued:
 $errorExample
 ```
 
+List of all existing wallet error tags: $walletErrorList.
```

Comment:
What about rendering these are a nice markdown table, like this? We could then, in the future, fill those `-`. For now don't worry about them:

```
| Error Name      | HTTP Error code | Example |
|-----------------|-----------------|---------|
| WalletNotFound  | -               | -       |
| MigrationFailed | -               | -       |
```

This would render as:

| Error Name      | HTTP Error code | Example |
|-----------------|-----------------|---------|
| WalletNotFound  | -               | -       |
| MigrationFailed | -               | -       |

Reply:
+1. Will fix it.

Reply:
@adinapoli-iohk would `Example` column show something like:
```


{
    "diagnostic": {},
    "message": "WalletNotFound"
}
``` 
or do you have another idea?

Reply:
Anyway, followup task here https://iohk.myjetbrains.com/youtrack/issue/CSL-2032

Reply:
![markdown error table](https://user-images.githubusercontent.com/2461951/33626236-45cb23c2-d9fa-11e7-9df5-0ab4009ac035.png)

Reply:
Lovely @akegalj ! I really hope we will be able to fill the other two colums with a bit of "type trickery" and generic programming 😀  (but this outside the scope of this ticket).

Reply:
I would like to try to tackle that one if there is time . I believe it is a non brainer for more experienced once. Hope nobody will pick up this task before I do one day

Reply:
@akegalj Feel free then to open a ticket, put in in the board in the "backlog" row, assign it to you and part it for now :)